src/logic/azext_logic/generated/_help.py

```diff
@@ -0,0 +1,119 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['logic workflow'] = """
+    type: group
+    short-summary: logic workflow
+"""
+
+helps['logic workflow list'] = """
+    type: command
+    short-summary: Gets a list of workflows by subscription.
+    examples:
+      - name: List all workflows in a resource group
+        text: |-
+               az logic workflow list --resource-group "test_resource_group"
+"""
+
+helps['logic workflow show'] = """
+    type: command
+    short-summary: Gets a workflow.
+    examples:
+      - name: Get a workflow
+        text: |-
+               az logic workflow show --resource-group "test_resource_group" --name "test_workflow"
+"""
+
+helps['logic workflow create'] = """
+    type: command
+    short-summary: Creates or updates a workflow using a JSON file for the defintion.
+    examples:
+      - name: Create or update a workflow
+        text: |-
+               az logic workflow create --resource-group "test_resource_group" --location "centralus" --name "test_workflow" --definition "workflow.json"
```

Comment:
There can be, but it would not be very helpful. Workflows are fairly complex and the logic service team does not even have a model backing a workflow definition, they just treat is as generic JObject on service side and this why the swagger has workflowsProperties as just a generic object. Moreover, this why the logic app design tools exist, it will generate the definition here for you, which is the suggested way the logic app service team want user to generate the JSON.

Reply:
Can you add this information in README? It seems this extension should be used together with the logic app design tools.

Reply:
Added